osf/models/analytics.py

```diff
@@ -105,7 +107,8 @@ def update_counter(cls, resource, file, version, action, node_info, session_obj)
         cleaned_page = cls.clean_page(page)
         date = timezone.now()
         date_string = date.strftime('%Y/%m/%d')
-        visited_by_date = session_obj.data.get('visited_by_date', {'date': date_string, 'pages': []})
+        session_obj = SessionStore(session_key=session_key)
```

Comment:
Comment for myself: since we are passing the key instead of the object, there is still a chance of failure to find the existing session. Need an `existence` check here too.